cdap-watchdog/src/main/java/co/cask/cdap/logging/read/AvroFileReader.java

```diff
@@ -181,9 +174,34 @@ public void readLog(Location file, Filter logFilter, long fromTimeMs, long toTim
     }
   }
 
+  private List<LogEvent> readToNextSync(Filter logFilter, long fromTimeMs, DataFileReader<GenericRecord> dataFileReader,
```

Comment:
You are right. We read in forward direction. Leave the method name as-is. Add javadoc to describe what the method does then.